modin/pandas/series.py

```diff
@@ -1251,7 +1251,18 @@ def unique(self):
         return self._query_compiler.unique().to_numpy().squeeze()
 
     def update(self, other):
-        return self._default_to_pandas(pandas.Series.update, other)
+        """
+        Modify Series in place using non-NA values from passed
+        Series. Aligns on index.
+
+        Parameters
+        ----------
+        other : Series, or object coercible into Series
+        """
+        if not isinstance(other, Series):
+            other = Series(other)
+        query_compiler = self._query_compiler.series_update(other._query_compiler,)
```

Comment:
Minor nit: remove trailing comma

```suggestion
        query_compiler = self._query_compiler.series_update(other._query_compiler)
```